CMakeLists.txt

```diff
@@ -12,13 +12,22 @@ set(hpxmp_sources
         src/intel_hpxMP.cpp
         src/hpx_runtime.cpp
         src/kmp_atomic.cpp
-        src/loop_schedule.cpp)
+        src/loop_schedule.cpp
+        src/ompt_hpx-general.cpp)
 
```

Comment:
Should we include all ompt sources regardless whether OMPT support was configuired?